common/src/main/java/com/graphicsfuzz/common/util/PipelineInfo.java

```diff
@@ -251,6 +251,35 @@ public void removeUniformBinding(String uniformName) {
     }
   }
 
+  public int appendValueToUniform(String uniformName, Number value) {
+
+    if (!dictionary.has(uniformName)) {
+      throw new IllegalArgumentException("Uniform declaration not found.");
+    }
+
+    dictionary.getAsJsonObject(uniformName).get("args").getAsJsonArray().add(value);
+
+    return dictionary.getAsJsonObject(uniformName).get("args").getAsJsonArray().size() - 1;
+  }
+
+  public int getUnusedBindingNumber() {
+    int number = 0;
```

Comment:
No point declaring this here - clearer to declare it in the header of the `for` loop below.

Reply:
Moved the declaration to the header of the loop.